===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(swReactorPoll_create:object)) at 25 (line 51, column 9) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/swoole-src/src/reactor/ReactorPoll.c
+++ b/table1/benchmarks/swoole-src/src/reactor/ReactorPoll.c
@@ -50,6 +50,7 @@ int swReactorPoll_create(swReactor *reactor, int max_fd_num)
     object->fds = sw_calloc(max_fd_num, sizeof(swPollFdInfo));
     if (object->fds == NULL)
     {
+        free(object);
         swWarn("malloc[1] failed");
         return SW_ERR;
     }
